packages/database/drizzle/meta/0000_snapshot.json

```diff
@@ -82,6 +82,64 @@
         }
       }
     },
+    "latitude.convos": {
+      "name": "convos",
+      "schema": "latitude",
+      "columns": {
+        "id": {
+          "name": "id",
+          "type": "bigserial",
+          "primaryKey": true,
+          "notNull": true
+        },
+        "uuid": {
+          "name": "uuid",
+          "type": "uuid",
+          "primaryKey": false,
+          "notNull": true,
+          "default": "gen_random_uuid()"
+        },
+        "content": {
+          "name": "content",
+          "type": "jsonb",
+          "primaryKey": false,
+          "notNull": false
+        },
```

Comment:
What's the `convo` content? Is it the list of messages?

We can save the convo's content as a latitude prompt, which also includes the convo's initial config.

Reply:
hmm it could lead to confusion if we use the prompt_snapshots table for both prompt templates and conversation histories

Reply:
Prompt files are representation of `Conversation` objects, which are just a list of messages.

What I mean is that the conversation history can be represented as another .prompt file, as it is just a list of messages too. The only difference is that they are already resolved, so there are no variables.

Reply:
But the compiled version is a json shape. The prompt is a text template no?